main/acle.md

```diff
@@ -4983,14 +4974,22 @@ function F if at least one of the following is true:
 Otherwise, ZA can be in any state on entry to A if at least one of the
 following is true:
 
-*   F [uses](#uses-state) `"za"`
+*   F [uses](#uses-state) `"za"`.
+
+*   F [uses](#uses-state) `"zt0"`.
+
+*   F is what the [[AAPCS64]](#AAPCS64) calls an ["agnostic-ZA"](#agnostic-za)
+    function and A's clobber-list does not include `"za"` or `"zt0"`.
```

Comment:
```suggestion
*   F's type has an [`__arm_agnostic("sme_za_state")`  attribute](#agnostic-za)
    and A's clobber-list includes neither `"za"` nor `"zt0"`.
```
so that we keep it in terms of C/C++-level constructs.